tuya.md

```diff
@@ -0,0 +1,288 @@
+# Supporting Tuya Devices
+
+> [!IMPORTANT]
+> The following should work for most Tuya devices. Some devices may require additional reverse engineering to unlock all functions.
```

Comment:
Ok, now that I actually look at it, maybe we don't really need the fancy markdown here.
Preview: https://github.com/zigpy/zha-device-handlers/blob/b4fc995f9833a0a173c6aac9310815f0fcf860aa/tuya.md

Reply:
eh, but maybe it's fine 😄